CHANGELOG.md

```diff
@@ -29,12 +33,15 @@
 * [ENHANCEMENT] Store-gateway: improved performance of series matching. #3391
 * [ENHANCEMENT] Move the validation of incoming series before the distributor's forwarding functionality, so that we don't forward invalid series. #3386 #3458
 * [ENHANCEMENT] S3 bucket configuration now validates that the endpoint does not have the bucket name prefix. #3414
+* [ENHANCEMENT] Store-gateway: index stats response. #3206
```

Comment:
What does this mean?

Reply:
We can rewrite it as:

```suggestion
* [ENHANCEMENT] Query-frontend: added "fetched index bytes" to query statistics, containing the total bytes read by store-gateways from TSDB block indexes. #3206
```

Reply:
> The term "containing" is ambiguous: is it "that contain" or "so that the statistics contain" or something else?

The latter, I updated it accordingly: 45b55e81d